app/serializers/concerns/case_mixin.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 module CaseMixin
+  HAS_SALESFORCE_CASE ||= "has_salesforce_case"
```

Comment:
```suggestion
  HAS_SALESFORCE_CASE = "has_salesforce_case"
```

^ should be enough, unless the file is required more than once (in such case, it should not be required more than once 😛)

Reply:
It's a force of habit to be honest ;)

Reply:
Ok, changed it.